### DISCUSSION TITLE
include path error

### DISCUSSION THREAD

**Discussion by sem1hs** [microsoft/vscode-cpptools#10134] [General]
since i update my mac m1 to ventura 13.0, i have this problem ;
<img width="1106" alt="error" src="https://user-images.githubusercontent.com/117932821/201157088-84f5748d-7319-4af8-9eae-8575abb8f471.png">

**Comment by sean-mcmanus** [comment]
Can you run the C/C++: Log Diagnostics command after opening apa.c and look what Include Paths are reported? It appears to be missing the include path for stdio.h.

**Reply by sem1hs** [reply]
which command do i will run ?

**Reply by sem1hs** [reply]
like this ?
<img width="1132" alt="Ekran Resmi 2022-11-11 19 14 17" src="https://user-images.githubusercontent.com/117932821/201383426-00ee2f2a-6781-4e48-a105-2cc110d3a3c7.png">

**Reply by sean-mcmanus** [reply]
No, it's a command in the VS Code View->Command Palette.

**Reply by sem1hs** [reply]
is this true ?
<img width="755" alt="Ekran Resmi 2022-11-11 22 06 35" src="https://user-images.githubusercontent.com/117932821/201412865-0acef43c-d91e-4953-947c-a202df184c3e.png">

**Reply by sem1hs** [reply]
ever since i updated my m1 mac ventura 13.0 and i am taking this problem

**Reply by sean-mcmanus** [reply]
Is what true?

**Reply by sem1hs** [reply]
photo

**Reply by sean-mcmanus** [reply]
I don't understand. What are you trying to show in the photo?

**Reply by sem1hs** [reply]
I solved this problem. I opened the terminal and xcode-select --install i ran this code so the problem solved.
